cpp/test/prims/silhouette_score.cu

```diff
@@ -207,7 +207,8 @@ class silhouetteScoreTest
 //setting test parameter values
 const std::vector<silhouetteScoreParam> inputs = {
   {4, 2, 3, raft::distance::DistanceType::L2Expanded, 4, 0.00001},
-  {4, 2, 2, raft::distance::DistanceType::L2SqrtUnexpanded, 2, 0.00001},
+  // FIXME: temporarily disabled due to flaky test
```

Comment:
If possible could you use `GTEST_SKIP` so we get a reminder of the number of disabled tests during test runs? (example https://github.com/rapidsai/cuml/blob/d5687ea5fa1f6cd3d6947ff32f495ee89f137895/cpp/test/sg/svc_test.cu#L1159)

Reply:
Not sure if it's doable, since the test is parameterized, and I'm disabling just a single parameterization.